__init__.py

```diff
@@ -24,16 +24,16 @@ def name():
 def description():
     return "Edit toolbar for PostGIS topology primitives (ISO SQL/MM based)"
 def version():
-    return "0.2.2"
+    return "0.3.1"
 def icon():
     return "icons/topoedit.png"
 def qgisMinimumVersion():
-    return "1.8"
+    return "3.0"
 def qgisMaximumVersion():
-    return "2.99"
+    return "3.99"
 def category():
     return "Database"
 def classFactory(iface):
     # load PgTopoEditor class from file PgTopoEditor
-    from pgtopoeditor import PgTopoEditor
+    from .pgtopoeditor import PgTopoEditor
```

Comment:
(why) is this required ?

Reply:
The plugin directory is not in the pythonpath so it could not import without the dot. So it's pgtopoeditor.pgtopoeditor or .pgtopoeditor but just pgtopoeditor imports the package. Then you'd have to move PgTopoEditor as an import to __init__.py, I think.